cdap-app-fabric/src/test/java/co/cask/cdap/AppWithSchedule.java

```diff
@@ -50,8 +50,11 @@ public void configure() {
       scheduleProperties.put("anotherKey", "anotherValue");
       scheduleProperties.put("someKey", "someValue");
 
-      scheduleWorkflow(Schedules.createTimeSchedule("SampleSchedule", "Sample schedule", "0/1 * * * * ?"),
-                       "SampleWorkflow", scheduleProperties);
+      scheduleWorkflow(Schedules.builder("SampleSchedule")
```

Comment:
That makes sense. But then you could argue that constraints (such as how many concurrent runs are allowed) could also be different for multiple programs on the same schedule...